lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1353,16 +1352,26 @@ StructuralEquivalenceContext::findUntaggedStructOrUnionIndex(RecordDecl *Anon) {
   return Index;
 }
 
-bool StructuralEquivalenceContext::IsStructurallyEquivalent(Decl *D1,
-                                                            Decl *D2) {
+bool StructuralEquivalenceContext::IsEquivalent(Decl *D1, Decl *D2) {
+
+  // Ensure that the implementation functions (all static functions in this TU)
```

Comment:
I think at least part of this text should go to the declaration because that is where most of the useful information is expected to be found regarding a function. So the potential users should know when they are not supposed to call this.

Reply:
Ok, updated with extra comments at the declaration in the header.